tensorrt_llm/_torch/modules/fused_moe/routing.py

```
@@ -650,160 +739,725 @@ def routing_method_type(self) -> RoutingMethodType:
+        target_experts = []
+
+        for token_idx, desired_gpus in enumerate(desired_gpu_timeline):
+            slot_groups = self._plan_token_groups(desired_gpus,
+                                                  group_gpu_experts,
+                                                  groups_by_gpu, topk_group,
+                                                  token_idx, ep_rank)
+            used_experts = set()
+            token_targets = []
+
+            for slot_idx, (desired_gpu, group_idx) in enumerate(
+                    zip(desired_gpus, slot_groups)):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)